scripts/install.ps1

```diff
@@ -0,0 +1,3 @@
+$version = (New-Object System.Net.WebClient).DownloadFile("https://download.newrelic.com/install/newrelic-cli/currentVersion.txt").Trim();
```

Comment:
@ctrombley we need to include this for the older windows versions
[Net.ServicePointManager]::SecurityProtocol = 'tls12, tls';

Reply:
👍